### PR TITLE
Add long help for switch replace command so we have some descripton o…

### PR DESCRIPTION
**Pull request by mreiger**
…f the replacement process

### CODE REVIEW

File: docs/metalctl_switch.md [hunk 1]
```diff
@@ -48,6 +48,6 @@ switch are the leaf switches in the data center that are controlled by metal-sta
 * [metalctl switch detail](metalctl_switch_detail.md)	 - switch details
 * [metalctl switch edit](metalctl_switch_edit.md)	 - edit the switch through an editor and update
 * [metalctl switch list](metalctl_switch_list.md)	 - list all switches
-* [metalctl switch replace](metalctl_switch_replace.md)	 - puts a switch in replace mode in preparation for physical replacement
+* [metalctl switch replace](metalctl_switch_replace.md)	 - Put a leaf switch into replace mode in preparation for physical replacement. For a description of the steps involved see the long help.
```
Comment (majst01) [inline docs/metalctl_switch.md:51]:
```suggestion
* [metalctl switch replace](metalctl_switch_replace.md)	 - put a leaf switch into replace mode in preparation for physical replacement. For a description of the steps involved see the long help.
```

**Review by majst01 — DISMISSED** [review]
OK from my side, one small nit. Maybe @mwennrich and Manfred should review as well

**Comment by mschuller** [comment]
That's a good description of what we did.

File: cmd/switch.go [hunk 2]
```diff
@@ -53,7 +53,20 @@ func newSwitchCmd(c *config) *cobra.Command {
...
 	switchReplaceCmd := &cobra.Command{
 		Use:   "replace <switchID>",
-		Short: "puts a switch in replace mode in preparation for physical replacement",
+		Short: "put a leaf switch into replace mode in preparation for physical replacement. For a description of the steps involved see the long help.",
+		Long: `Put a leaf switch into replace mode in preparation for physical replacement
+
+Operational steps to replace a switch:
+
+- Put the switch that needs to be replaced in replace mode with this command
+- Replace the switch MAC address in the metal-deployment
```
Comment (Gerrit91) [inline cmd/switch.go:62]:
```suggestion
- Replace the switch MAC address in the metal-stack deployment configuration
```

File: cmd/switch.go [hunk 3]
```diff
@@ -53,7 +53,20 @@ func newSwitchCmd(c *config) *cobra.Command {
...
+- Make sure that interfaces on the new switch do not get connected to the PXE-bridge immediately by setting the interfaces dictionary of the respective leaf switch to [] in the metal-deployment repository
```
Comment (Gerrit91) [inline cmd/switch.go:63]:
```suggestion
- Make sure that interfaces on the new switch do not get connected to the PXE-bridge immediately by setting the interfaces list of the respective leaf switch to [] in the metal-stack deployment configuration
```

File: cmd/switch.go [hunk 4]
```diff
@@ -53,7 +53,20 @@ func newSwitchCmd(c *config) *cobra.Command {
...
+- Deploy the management servers so that the dhcp servers will serve the right address and DHCP options to the new switch
+- Replace the switch physically. Be careful to ensure that the cabling mirrors the remaining leaf exactly because the new switch information will be cloned from the remaining switch! Also make sure to have console access to the switch so you can start and monitor the install process
+- If the switch is not in onie install mode but already has an operating system installed, put it into install mode with "sudo onie-select -i -f -v" and reboot it. Now the switch should be provisioned with a management IP from a management server, install itself with the right software image and receive license and ssh keys through ZTP. You can check whether that process has completed successfully with the command "sudo ztp -s". The ZTP state should be disabled and the result should be success.
+- Install the switch with the "bake <partition>" CI job
+- Deploy metal-core on the switch with the "deploy <partition>" CI job
```
Comment (Gerrit91) [inline cmd/switch.go:68]:
```suggestion
- Deploy the switch plane and metal-core through metal-stack deployment CI job
```

**Review by Gerrit91 — CHANGES_REQUESTED** [review]
There's few parts which should be written in a bit more general manner because deployment is described differently in metal-stack docs. Remember this is an open source project, so it should be aligned to a certain degree. :)

**Pull request merged**